app/main.py

```diff
@@ -1,5 +1,69 @@
+from __future__ import annotations
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
+        self.weight = weight
+
+
+class BaseRobot(Cargo):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] = None) -> None:
```

Comment:
coords could be both types, so use annotation like this. The same in all other similar cases

```suggestion
class BaseRobot(Cargo):
    def __init__(self,
                 name: str,
                 weight: int,
                 coords: list[int] | None = None) -> None:
```